packages/playwright/src/errorContext.ts

```diff
@@ -91,7 +91,7 @@ export async function attachErrorContext(testInfo: TestInfoImpl, format: 'markdo
 
     const parsedError = error.stack ? parseErrorStack(error.stack, path.sep) : undefined;
     const inlineMessage = stripAnsiEscapes(parsedError?.message || error.message || '').split('\n')[0];
-    const location = parsedError?.location || { file: testInfo.file, line: testInfo.line, column: testInfo.column };
+    const location = (parsedError?.location && await existsAsync(parsedError.location.file)) ? parsedError.location : { file: testInfo.file, line: testInfo.line, column: testInfo.column };
```

Comment:
This is prone to race conditions and probably `EPERM` as well. Let's make it a try/catch inside `loadSource` instead?